### ISSUE TITLE
How can these examples be converted to Swift for iOS

### ISSUE THREAD

**Issue by doleksa** [google/google-api-objectivec-client-for-rest#234]
I've tried the Calendar Sample on my Mac and it works great, but I would like to see the Calendar Sample in Swift so I could implement it in an iPhone or iPad app in iOS.  Can this be done?  Is it as simple as converting each file in the Calendar Sample to Swift code?

**Comment by thomasvl** [comment]
Yup, if you'd like to take the time and submit a new example written in Swift we'd be happen to accept it.

**Comment by kent7301** [comment]
Общество всемирных межбанковских финансовых каналов связи

**Comment by rpstro02** [comment]
I'm considering submitting a new example app with the Sheets api in Swift. Are you still accepting pull requests?

**Comment by thomasvl** [comment]
Yup, happy to take a look at a PR to add one!

**Comment by vonsamnang01041994** [comment]
👍👏

**Comment by vonsamnang01041994** [comment]
[](https://apps.apple.com/hk/app/id1492241188)

**Comment by waterdrake** [comment]
Swift examples on how to execute queries and set things up in an iOS app would be amazing.

**Comment by zahracarthage** [comment]
was this ever updated ?

**Comment by thomasvl** [comment]
Sadly, no one has contributed any PRs to convert one of the existing examples or to write a new one.

**Comment by Volodymyr-13** [comment]
I can’t believe that Google is so poor that on this official repo there is no examples in Swift since 2018.
It’s 2023 omg.

It this some kind of community’s pet project? More looks like some kind of minimal support / abandoned project.

**Comment by thomasvl** [comment]
> I can’t believe that Google is so poor that on this official repo there is no examples in Swift since 2018. It’s 2023 omg.
> 
> It this some kind of community’s pet project? More looks like some kind of minimal support / abandoned project.

Your tone isn't going to exactly get you a lot of help.

The this project is supported, we regularly update for new service definitions and to address any issues that come up. As with opensource projects, it is up to the community to also help providing things, and so far no one has taken the time to provide a Swift based example, but there are lots of works working with the library via their own Swift.

**Comment by waterdrake** [comment]
> > I can’t believe that Google is so poor that on this official repo there is no examples in Swift since 2018. It’s 2023 omg.
> > It this some kind of community’s pet project? More looks like some kind of minimal support / abandoned project.
> 
> Your tone isn't going to exactly get you a lot of help.
> 
> The this project is supported, we regularly update for new service definitions and to address any issues that come up. As with opensource projects, it is up to the community to also help providing things, and so far no one has taken the time to provide a Swift based example, but there are lots of works working with the library via their own Swift.

I agree with you, the tone does not help.

I think the people post frustrated/angry are doing so because they (somewhat rightfully) view themselves as paying Google Cloud customers. Libraries such as these are expected to be up-to-date because without them, using those Google Cloud services is frustrating.

The question is, where is the line between what Google is expected to provide as a paid service in their Google Cloud products versus what is open source.

I would argue libraries to use core Google Cloud services that are advertised front and center on their website should be up to date. After all, customers (e.g. companies, developers) invest a lot of time into learning these services, using them and then ultimately paying thousands of dollars a month+ for them.

Still, I agree, the tone does not help. But I understand the frustration from the point of view as a customer - to which the OP seems to be.

**Comment by Gerlie123** [comment]
Help me Everyone no idea of this
